ansible_collections/arista/avd/roles/eos_designs/templates/facts/switch/switch.j2

```diff
@@ -187,6 +187,19 @@ switch:
                           switch.default_evpn_role, 'none') %}
   evpn_role: {{ switch_evpn_role }}
 
+{# switch.evpn_gateway #}
+{%     set switch_evpn_gateway = switch_data.combined.evpn_gateway.evpn_l2.enabled is arista.avd.defined(true) or
+                                 switch_data.combined.evpn_gateway.evpn_l3.enabled is arista.avd.defined(true) or
+                                 switch_data.combined.evpn_gateway.vpn_ipv4.enabled is arista.avd.defined(true) %}
+  evpn_gateway: {{ switch_evpn_gateway | arista.avd.default(false) }}
+  evpn_gateway_evpn_l2: {{ switch_data.combined.evpn_gateway.evpn_l2.enabled | arista.avd.default(false) }}
```

Comment:
Not sure "evpn_l2/l3" are the best names for this since there may be evpn-mpls l2 and l3 gateway features in the future. Maybe vxlan_l2/l3 or vxlan_l2_dci/vxlan_l3_dci or something is better.

Reply:
Changed to evpn_gateway_vxlan_l2/l3